app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestFragment.java

```diff
@@ -9,6 +9,7 @@
 import io.intrepid.contest.R;
 import io.intrepid.contest.base.BaseFragment;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.customviews.HintLabelEditText;
```

Comment:
You should actually keep `EditText` because `HintLabelEditText` is the linear layout surrounding it (I realize now the name is confusing!). That will change when I refactor `HintLabelEditText`.